data_loader.py

```
@@ -0,0 +1,24 @@
+"""
+Use to give data to alpha
+"""
+import pickle
+import os
+
+class DataLoader():
+    """
+    load the data which users' need
+    """
+    def __init__(self, data_list):
+        self.data = {}
+        for data_name in data_list:
+            if os.path.isdir(data_name):
+                with open(data_name, "rb") as data:
+                    self.data[data_name] = pickle.load(data)
+            else:
+                print("Error in DataLoader.")
```

I prefer to raise an exception to interrupt current process instead of printing out an error message and doing nothing else. I guess you don't have any experience about exception handling. If so, I strongly suggest you to [study it](https://docs.python.org/3/tutorial/errors.html) for now as you are ready to touch more system issues.